src/components/scenes/GettingStartedScene.tsx

```diff
@@ -394,7 +404,7 @@ const SectionCoverAnimated = styled(Animated.View)<{ swipeOffset: SharedValue<nu
       justifyContent: 'space-between',
       backgroundColor: '#0F1D26',
       paddingVertical: theme.rem(1),
-      paddingBottom: insets.bottom,
+      paddingBottom: insets.bottom + themeRem,
```

Comment:
Please use `theme.rem(1)`, like on the line above. We use the `themeRem` in animated props because we can't call methods across the reanimated bridge, so `theme.rem()` is unusable. Otherwise prefer the normal `theme.rem(1)` syntax.